### ISSUE TITLE
Respect `package.json#sideEffects` and skip those files

### ISSUE THREAD

**Issue by matzeeable** [lukastaegert/eslint-plugin-tree-shaking#136]
Hey!

In webpack, it is possible to mark files which allow side effects via [`package.json#sideEffects`](https://webpack.js.org/guides/tree-shaking/).

At first try, I figured out that this plugin does not respect this configuration.

What do you think about this? 😊

**Comment by lukastaegert** [comment]
If you think there is a reasonable way to incorporate this, PRs are usually welcome.

**Comment by matzeeable** [comment]
Yeah, this would be cool and I could create a PR for this. But do you know a way in ESLint to get the projects' `package.json` path?
